tests/com/goide/inspections/GoHighlightingTest.java

```diff
@@ -276,7 +278,7 @@ public void testPackageWithTestPrefix() throws Throwable {
       public VirtualFile compute() throws Throwable {
         myFixture.getTempDirFixture().createFile("pack1/pack1_test.go", "package pack1_test; func Test() {}");
         return myFixture.getTempDirFixture().createFile("pack2/pack2_test.go",
-                                                        "package pack2_test; import \"testing\"; func TestTest(t *testing.T) {<error>pack1_test</error>.Test()}");
+                                                        "package pack2_test; import \"testing\"; func TestTest(*testing.T) {<error>pack1_test</error>.Test()}");
```

Comment:
I have to do the change, either like this or to declare the parameter as `_`, otherwise this would fail the test.